### PR TITLE
docs(spinnaker): add Xiaohongshu to user list

### PR DESCRIPTION
**Pull request by thomassun**
Hi, this is a E-Commerce in China, our homepage is https://www.xiaohongshu.com/ , we are now using spinnaker @kubernetes as the main CD tool,
if we may, We'd like to be listed in https://www.spinnaker.io/community/#whos-using-spinnaker.

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -91,6 +91,9 @@ grubhub_user:
 contaazul_user:
   image_path: assets/images/users/contaazul_logo.svg
   alt: "ContaAzul Logo"
+xiaohongshu_user:
+  image_path: assets/images/users/xiaohongshu_logo.svg
```
Comment (skim1420) [inline community/index.md]: I believe this should be jpg, not svg.

Comment (skim1420) [reply community/index.md]: Also, the image file should be placed in assets/images/users/

**Review by skim1420 — CHANGES_REQUESTED** [review]
Welcome to the community! Please fix the mentioned errors, thanks.

File: community/index.md (hunk 1 above)
Comment (dorbin) [reply community/index.md]: @skim1420 I think we have both. And svg scales better, right? Do we need to require png?

Comment (dorbin) [reply community/index.md]: Ah I see. The image you added, @thomassun is a jpg, but you referenced it as an svg. @skim1420 is right, should be jpg.

If you have an svg version, that would be great. Or  a png would also be better than jpg. However jpg is acceptable.

**Review by skim1420 — APPROVED** [review]

**Pull request merged**